libvast/test/parseable.cpp

```diff
@@ -437,14 +437,22 @@ TEST(signed integral with digit constraints) {
   CHECK(!p("-1"));
   CHECK(p("-1234", x));
   CHECK_EQUAL(x, -1234);
-  MESSAGE("partial match");
+  MESSAGE("partial match with additional digit");
   auto str = "12345"sv;
   auto f = str.begin();
   auto l = str.end();
   CHECK(p.parse(f, l, x));
   REQUIRE(f + 1 == l);
   CHECK_EQUAL(*f, '5');
   CHECK_EQUAL(x, 1234);
+  MESSAGE("partial match with non-digits character");
+  str = "6789x"sv;
```

Comment:
I think this test should cover the case of an integral with less than `MaxDigits` digits. So `"678x"sv` would make more sense to me.